flow360/component/resource_base.py

```diff
@@ -97,6 +97,15 @@ def handle_none_str(cls, value):
             value = None
         return value
 
+    @property
+    def category_tag(self):
+        """
+        Returns the first tag.
+        """
+        if self.tags:
+            return self.tags[0]
+        return None
+
```

Comment:
I suggest to remove concept of `category_tag` and in report allow user to chose by which tag to group, 1st being the default.